File: testprojects/c#/presentation/gui/orkut/AccountGUI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.orkut;

namespace CSharpBenchmark.presentation.gui.orkut
{
    //Functional requirement 3.1.2
    //Test case 65: Only class presentation.gui.orkut.AccountGUI may have a dependency with domain.orkut.Account 
    //Result: TRUE
    public class AccountGUI
    {
        public AccountGUI()
        {
            //FR5.1
            new Account();
        }
    }
}

File: testresources/java/benchmark_2012/presentation/gui/facebook/AccountGUI.java
package presentation.gui.facebook;

import domain.facebook.Account;

//Functional requirement 3.1.2
//Test case 66: Only class presentation.gui.facebook.AccountGUI may have a dependency with class domain.Facebook.Account
//Result: FALSE
public class AccountGUI {
	public AccountGUI(){
		//FR5.1
		new Account();
	}
}